README.md

```diff
@@ -4,11 +4,23 @@ PyGYW is a Python library that provides an easy-to-use interface to control the
 
 ## Installation
 
-To install PyGYW, clone the repository and use pip:
+To install PyGYW, append this line to your `requirements.txt`:
 
-```console
-> git clone https://www.github.com/getyourway/pygyw
-> pip install -e pygyw
+`pygyw @ git+https://github.com/getyourway/pygyw.git`
+
+Then run the following:
+
+```shell
+pip install --upgrade pip
+pip install -r requirements.txt
+```
+
+If you are a GYW employee and you want to develop PyGYW, clone the develop branch through SSH and install it in editable mode:
```

Comment:
This is very good explanation, just it is not limited to GYW developers, but to anyone who wants to contribute
```suggestion
If you are a developer and you want to contribute to PyGYW, clone the develop branch through SSH and install it in editable mode:
```